inventory/service/group_vars/apimon.yaml

```diff
@@ -353,6 +353,50 @@ apimon_instances:
       - name: production_eu-de-csm
         cloud: otcapimon_csm1
 
+  production_2:
+    statsd_host: 127.0.0.1
+    gearman_host: apimon-scheduler-prod-2
+    zone: production_eu-de_2
+
+    # Endpoint monitoring configuration
+    # (inventory/service/group_vars/apimon-epmon.yaml)
+    epmon_clouds:
+      - production_eu-de_2:
```

Comment:
we should still use "production_eu-de" to keep metrics in the DB properly - we do not really want to introduce a yet another dropdown value to watch